modules/contentbox/models/BaseEntityMethods.cfc

```diff
@@ -8,32 +8,45 @@
 */
 component mappedsuperclass="true"{
 
-	// PK Pointer			
+	// PK Pointer
 	this.pk = "PLEASE_SELECT_ONE";
 	// Constraints Default
 	this.constraints = {};
 
 	/* *********************************************************************
-	**						PUBLIC FUNCTIONS								
+	**						PUBLIC FUNCTIONS
 	********************************************************************* */
 
 	/**
 	* Constructor
 	*/
 	function init(){
+		variables.ORMUtil = new cborm.models.util.ORMUtilFactory().getORMUtil();
+
+		var uuidLib = createobject("java", "java.util.UUID");
+
+		var keyColumn = getKey();
```

Comment:
I would prefer to stay away from metadata calls for just the name of the primary key.  That's why there is a `this.pk` that each entity fills out to avoid the expensive nature of metdata.

Reply:
I'll refactor over and see if it handles the needs of the component.